python/pyarrow/tensorflow/plasma_op.cc

```diff
@@ -157,11 +157,14 @@ class TensorToPlasmaOp : public tf::AsyncOpKernel {
 
     uint8_t* data = reinterpret_cast<uint8_t*>(data_buffer->mutable_data() + offset);
 
-    auto wrapped_callback = [this, context, done, data_buffer, object_id]() {
+    auto wrapped_callback = [this, context, done, data_buffer, data, object_id]() {
       {
         tf::mutex_lock lock(mu_);
         ARROW_CHECK_OK(client_.Seal(object_id));
         ARROW_CHECK_OK(client_.Release(object_id));
+        auto orig_stream = context->op_device_context()->stream();
+        auto stream_executor = orig_stream->parent();
+        CHECK(stream_executor->HostMemoryUnregister(static_cast<void*>(data)));
```

Comment:
We probably only want this to happen in the GPU case, right?

Reply:
Same with the other op.

Reply:
Yeah, I updated it (it actually doesn't compile for cpu only tensorflow).